e2e-tests/spring-rest-openapi-v3/src/main/kotlin/com/foo/rest/examples/spring/openapi/v3/authenticatedswaggerendpoint/AuthenticatedSwaggerEndpointRest.kt

```diff
@@ -0,0 +1,54 @@
+package com.foo.rest.examples.spring.openapi.v3.authenticatedswaggerendpoint
+
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.io.File
+
+
+@RestController
+class AuthenticatedSwaggerEndpointRest {
+
+    private val SECRET = "Secret Authentication Bearer..."
+
+    // OD - Note, this path will work for e2e test but not for manual test
+    // For manual testing, the working directory should be set to ./e2e-tests/spring-rest-openapi-v3/
+    private val SWAGGER_PATH = "./src/main/resources/static/authenticatedSwaggerEndpointOpenAPIv3.json"
```

Comment:
there is an easier, more robust way to deal with this... you can load it as a resource:

`this::class.java.classLoader.getResource(SWAGGER_PATH).readText()`

where `SWAGGER_PATH="/static/authenticatedSwaggerEndpointOpenAPIv3.json"`

Reply:
this will avoid issues in setting the current working directory

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one. It works both in EM tests and also when I test manually using Postman.